modules/get-started/pages/licensing.adoc

```diff
@@ -1,17 +1,13 @@
 = Enterprise Licensing 
 :description: Learn how to purchase and apply an Enterprise Edition license.
 
-Anyone can use the Certified and Community connectors, but you need an Enterprise Edition license to use Enterprise connectors.
+Anyone can use the Certified and Community connectors, but you need a valid Enterprise Edition license to use Enterprise connectors. 
 
-== Purchase an Enterprise Edition license
-
-You can purchase and download a license from the https://www.redpanda.com/upgrade[Redpanda website^].
+You can either https://www.redpanda.com/upgrade[upgrade to an Enterprise Edition license^], or http://redpanda.com/try-enterprise[generate a trial license key^] that's valid for 30 days.
 
 == Apply a license key to Redpanda Connect
 
-Redpanda Connect checks the license key status at runtime. If the license key is unavailable or has expired, you are blocked from using Enterprise connectors. You can view the license key’s expiration date at any time in the Redpanda Connect logs.
-
-Once you have downloaded a license key, you can apply it in the following ways: 
+Once you have downloaded the license key, you can apply it in the following ways: 
```

Comment:
```suggestion
When you have downloaded the license key, you can apply it in the following ways: 
```

Reply:
https://github.com/redpanda-data/docs-site/blob/main/meta-docs/STYLE-GUIDE.adoc#do-not-use-word-list

Reply:
Updated